staging/istio/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: istio
-version: 1.6.9
+version: 1.6.10
```

Comment:
I've been trying to keep the chart and app version aligned. How would you feel about folding in a patch of the app into this PR? yay/nay?